appveyor.yml

```diff
@@ -14,15 +14,27 @@ install:
   - cmd: SET PATH=C:\sbt\sbt\bin;%JAVA_HOME%\bin;%PATH%
   - cmd: SET SBT_OPTS=-XX:MaxPermSize=2g -Xmx4g
```

Comment:
feel free to take our config https://github.com/ensime/ensime-server/blob/2.0/appveyor.yml